src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Revisions.cshtml

```diff
@@ -5,13 +5,17 @@
     ViewData["Title"] = "Revisions";
 }
 
-<div class="row" asp-resource="@Model.Review">
-    <div class="col">
-        <div class="mt-2">
-            <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#uploadModel">Add revision</button>
+@if (!Model.Review.IsLocked)
+{
+    <div class="row" asp-resource="@Model.Review">
```

Comment:
What about other operations like deleting or renaming a revision, closing, deleting the review?

Reply:
Consider creating an auth policy and using it along with `asp-resource,asp-requirement`

Reply:
Delete and close are only allowed for owner user. right?  I didn't add that check since it is going to be created by bot user. I can definitely make this change if you think it's better to keep it uniform for all checks.

Reply:
Hm, true. Still, let's use an auth policy to control the automatic review.